frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.unit.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { ORDERS, SAMPLE_DATABASE } from "__support__/sample_database_fixture";
+import StructuredQuery from "metabase-lib/queries/StructuredQuery";
+import ExpressionWidget, { ExpressionWidgetProps } from "./ExpressionWidget";
+
+describe("ExpressionWidget", () => {
+  it("should render proper controls", () => {
+    setup();
+
+    expect(screen.getByText("Expression")).toBeInTheDocument();
+    expect(screen.getByText("Name")).toBeInTheDocument();
+
+    expect(screen.getByText("Cancel")).toBeInTheDocument();
+    expect(screen.getByText("Done")).toBeInTheDocument();
+  });
+
+  it("should render help icon with tooltip", () => {
+    setup();
+
+    const iconEl = screen.getByRole("img");
+    expect(iconEl).toBeInTheDocument();
+    expect(iconEl).toHaveClass("Icon-info");
+
+    userEvent.hover(iconEl);
+
+    expect(
+      screen.getByText(
+        "You can reference columns here in functions or equations, like: floor([Price] - [Discount]).",
+      ),
+    ).toBeInTheDocument();
+  });
+});
+
+function setup(additionalProps?: Partial<ExpressionWidgetProps>) {
```

Comment:
nit: I think a more common pattern across Metabase is to define default props when restructuring an argument:

```tsx
function setup({
  reportTimezone = "UTC",
  onChangeExpression = jest.fn(),
  ...props
}: Partial<ExpressionWidgetProps>) {
  ...
}
```